homeassistant/components/plugwise/config_flow.py

```diff
@@ -0,0 +1,83 @@
+"""Config flow for Plugwise integration."""
+import logging
+from typing import Any, Dict
+
+from Plugwise_Smile.Smile import Smile
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def _get_config_schema(input_dict: Dict[str, Any] = None) -> vol.Schema:
+    """
+    Return schema defaults for init step based on user input/config dict.
+
+    Retain info already provided for future form views by setting them as defaults in schema.
+    """
+    if input_dict is None:
+        input_dict = {}
+
+    return vol.Schema({vol.Required("host"): str, vol.Required("password"): str})
```

Comment:
Nothing custom is being built here.  You can define this as a constant above and get rid of this function `DATA_SCHEMA 
 = vol.Schema({vol.Required(CONF_HOST): str, vol.Required(CONF_PASSWORD): str})`